src/templates/base.html

```diff
@@ -13,13 +13,43 @@
     {% block head %}
       <title>{% block title %}The Web Almanac{% endblock %}</title>
       <script async src="https://www.googletagmanager.com/gtag/js?id=UA-22381566-3" nonce="{{ csp_nonce() }}"></script>
+      <script async src="/static/js/web-vitals.js" ></script>
       <script nonce="{{ csp_nonce() }}">
           window.dataLayer = window.dataLayer || [];
           function gtag() {
               dataLayer.push(arguments);
           }
           gtag('js', new Date());
           gtag('config', 'UA-22381566-3');
+
+          function sendWebVitalsGAEvents(metric) {
+            let event_value = 0;
```

Comment:
Actually one more comment. Can we swap `let` for `var`? We try to avoid ES6 syntax to gracefully degrade rather than throw an error on any old or non-standard browsers. Obviously they won't support web-vitals but this could still cause an error.

```suggestion
            var event_value = 0;
```

Reply:
Sure, forgot to change it after moving away from nodejs code.